src/github/schemas.ts

```diff
@@ -38,19 +38,25 @@ export const GitHubRepositorySchema = z.object({
   default_branch: z.string(),
 });
 
+
+const GitHubFileContentLinks = z.object({
+  self: z.string(),
+  git: z.number(),
+  html: z.string(),
+});
+
 // File content schemas
 export const GitHubFileContentSchema = z.object({
   type: z.string(),
-  encoding: z.string(),
   size: z.number(),
   name: z.string(),
   path: z.string(),
-  content: z.string(),
```

Comment:
This field is documented here: https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#get-repository-content

Can you please restore it?

Reply:
It could be made optional, though.